crossbeam-queue/Cargo.toml

```diff
@@ -15,6 +15,14 @@ description = "Concurrent queues"
 keywords = ["queue", "mpmc", "lock-free", "producer", "consumer"]
 categories = ["concurrency", "data-structures"]
 
+[features]
+default = ["std"]
+nightly = ["crossbeam-utils/nightly"]
```

Comment:
If you're adding a `nightly` feature to `crossbeam-queue`, I believe you should modify `ci/crossbeam-queue.sh` to match the structure of e.g. `ci/crossbeam-epoch.sh` wrt. handling features and `TRAVIS_RUST_VERSION`.

Reply:
Sure. I don't know whether it necessarily needs it - I just copied the setup from crossbeam-epoch.

Reply:
It would be good to test `crossbeam-queue` without the default features and with the `nightly` feature, just in case we add something to the crate that does require `std` or `nightly` :)

So you need to change the contents of `ci/crossbeam-queue.sh` to the following:

```bash
#!/bin/bash

cd "$(dirname "$0")"/../crossbeam-queue
set -ex

export RUSTFLAGS="-D warnings"

cargo check --no-default-features
cargo check --bins --examples --tests
cargo test

if [[ "$TRAVIS_RUST_VERSION" == "nightly" ]]; then
    cargo check --no-default-features --features nightly
    cargo test --features nightly
fi
```